### DISCUSSION TITLE
現在Qiitaでベータ版フィードバックが受信できない不具合が発生しております

### DISCUSSION THREAD

**Discussion by kskwtnk** [increments/qiita-discussions#245] [アナウンス]
現在Qiitaでベータ版フィードバックが受信できない不具合が発生しております。
詳しい原因等は現在調査中です。
対応が完了しましたら、再度こちらでアナウンスいたします。

お急ぎの場合はこちらからお願いします。
https://support.qiita.com/hc/ja/requests/new

ご迷惑をおかけして申し訳ございません。

**Comment by kyntk** [comment]
こちらの不具合について修正が完了しました。

Qiitaで [ベータ版フィードバック](https://qiita.com/settings/beta-release) から送信していただいたメッセージが、正常に受信されない不具合が発生しておりました。現在は復旧しております。
発見と対応が遅くなり申し訳ございません。フィードバックを送っていただいた方に、ご迷惑をおかけしましたことを、深くお詫び申し上げます。

## 不具合のあった期間
6月1日14時頃～ 7月5日 9時45分頃

## お願い
該当期間に、ベータ版フィードバックをお送りいただいている場合、メッセージを受信できていない可能性がございます。
大変恐れ入りますが、再度お送りいただけますと幸いです。
ご面倒をお掛けいたしますこと、重ねてお詫び申上げます。
